common/test/fixtures/enzyme-helpers.js

```diff
@@ -15,6 +20,28 @@ export function shallowWithTheme(Component: Node) {
   );
 }
 
+function themeAndContexts(Component: Node) {
+  return (
+    <ThemeProvider theme={theme}>
+      <OpeningTimesContext.Provider value={openingTimes}>
```

Comment:
These were just enough `Context`s to make whats-on render. I'm guessing we'll need to add more if/when we add tests to different pages (and if we need any of them to be configured, instead of taking fixed data).

Reply:
Should we just be using the one `mount` though across all tests?

Reply:
Possibly. What might be the pros/cons?

Reply:
Pros - it's more like the app.
Cons - a wee bit of refactoring.

Reply:
Sounds like on balance the pros should have it.

Reply:
separate pr, presumably?